packages/ui5-library-inquirer/package.json

```diff
@@ -40,7 +40,7 @@
     },
     "devDependencies": {
         "@types/inquirer-autocomplete-prompt": "2.0.1",
-        "@types/inquirer": "8.2.7"
+        "@types/inquirer": "8.2.6"
```

Comment:
I am curious, why did you switch to an older version of the types?

Reply:
It was misaligned wit the version in use : https://github.com/SAP/open-ux-tools/pull/1456/files/0ed37512b559b6d6f2af4216749e62c5a2f2926c#diff-4b021372a31ed3cd94bdc75d23e3bf011cba40e39479b52518989610f16eb8f2R38

Reply:
It was misaligned with the inquirer version being used.